website/content/docs/agent/template.mdx

```diff
@@ -80,50 +142,13 @@ The top level `template` block has multiple configurations entries:
 - `wait` `(object: required)` - This is the `minimum(:maximum)` to wait before rendering
   a new template to disk and triggering a command, separated by a colon (`:`).
 
-## Example Template
-
-Template with Vault Agent requires the use of the `secret` [function from Consul
-Template](https://github.com/hashicorp/consul-template/blob/master/docs/templating-language.md#secret).
-The following is an example of a template that retrieves a generic secret from Vault's
-KV store:
-
-```
-{{ with secret "secret/my-secret" }}
-{{ .Data.data.foo }}
-{{ end }}
-```
 
-## Example Configuration
+### Example
 
 The following demonstrates configuring Vault Agent to template secrets using the
```

Comment:
I think this paragraph no longer applies to this example.

Reply:
Good catch!